internal/builders/bazel/README.md

```diff
@@ -0,0 +1,252 @@
+# Generation of SLSA3+ provenance for Bazel builds
+
+This document explains how to generate SLSA provenance for Bazel artifact(s).
+
+This can be done by adding a step to your Github Actions workflow to call a
+[reusable
+workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows)
+to build the artifact(s) and generate SLSA provenance. We'll call this
+workflow the "Bazel builder" from now on.
+
+---
+
+<!-- markdown-toc --bullets="-" -i README.md -->
+
+<!-- toc -->
+
+- [Benefits of Provenance](#benefits-of-provenance)
+- [Development status](#development-status)
+- [Generating Provenance](#generating-provenance)
+  - [Getting Started](#getting-started)
+  - [Referencing the Bazel builder](#referencing-the-bazel-builder)
+  - [Private Repositories](#private-repositories)
+  - [Supported Triggers](#supported-triggers)
+  - [Workflow Inputs](#workflow-inputs)
+  - [Workflow Outputs](#workflow-outputs)
+  - [Provenance Format](#provenance-format)
+  - [Provenance Example](#provenance-example)
+
+<!-- tocstop -->
+
+---
+
+## Benefits of Provenance
+
+Using the Bazel builder will generate a non-forgeable attestation to the
+Bazel build artifacts produced using the identity of the GitHub workflow. This can be used to
+create a positive attestation to an artifact coming from your repository.
+
+That means that once your users verify the artifact they have downloaded they can
+be sure that it was created by your repository's workflow and hasn't been
+tampered with.
+
+## Development status
+
+The Bazel builder is currently in beta. The API could change while approaching
```

Comment:
Is this in `beta`? or are we planning to release it very soon?

If not we may want to just say it's under development.

Reply:
Right! Will change to say it's under development, thank you for the catch!